packages/airnode-utilities/src/gas-prices/gas-prices.ts

```diff
@@ -1,15 +1,21 @@
 import { BigNumber, ethers } from 'ethers';
-import { go, logger, PendingLog } from '@api3/airnode-utilities';
-import { BASE_FEE_MULTIPLIER, DEFAULT_RETRY_TIMEOUT_MS, PRIORITY_FEE } from '../constants';
-import { GasTarget, LogsData } from '../types';
-import { ChainOptions, PriorityFee } from '../config/types';
+import { ChainOptions, PriorityFee, GasTarget } from './types';
+import { go, logger, PendingLog } from '../';
+import { LogsData } from '../logging';
+
+// The default amount of time before a "retryable" promise is timed out and retried
+export const DEFAULT_RETRY_TIMEOUT_MS = 5_000;
+// The Priority Fee in Wei
+export const PRIORITY_FEE = 3120000000;
```

Comment:
Maybe better to include the `_IN_WEI` in the name to help anyone implementing this?
```suggestion
export const PRIORITY_FEE_IN_WEI = 3120000000;
```